examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/NetworkCredentialsParcelable.kt

```diff
@@ -0,0 +1,149 @@
+/*
+ *   Copyright (c) 2023 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package com.google.chip.chiptool
+
+import android.os.Parcelable
+import android.os.Parcel
+import android.os.Parcelable.Creator
+
+/** Class for holding WiFi or Thread credentials, but not both.  */
+class NetworkCredentialsParcelable : Parcelable {
+  var wiFiCredentials: WiFiCredentials?
+    private set
+  var threadCredentials: ThreadCredentials?
+    private set
+
+  private constructor(
+    wifiCredentials: WiFiCredentials?, threadCredentials: ThreadCredentials?
+  ) {
+    wiFiCredentials = wifiCredentials
```

Comment:
casing changed from java, meaning now there is no need for this here, but it seems confusing. Change casing back to `wifiCredentials` ?

Reply:
there is no casting here, this is set member variable from input parameter with the same name, changed to 'this.wiFiCredentials = wifiCredentials' to avoid confusion